memory_core/memory_core_magma.py

```diff
@@ -57,6 +57,51 @@ def lift_mem_core_ports(port, tile, tile_core):  # pragma: nocover
     tile.wire(tile.ports[port], tile_core.ports[port])
 
 
+def get_pond(use_sram_stub=1):
+    return MemCore(data_width=16,  # CGRA Params
+                   mem_width=16,
+                   mem_depth=32,
+                   banks=1,
+                   input_iterator_support=2,  # Addr Controllers
+                   output_iterator_support=2,
+                   input_config_width=16,
+                   output_config_width=16,
+                   interconnect_input_ports=2,  # Connection to int
+                   interconnect_output_ports=2,
+                   mem_input_ports=1,
+                   mem_output_ports=1,
+                   use_sram_stub=use_sram_stub,
+                   sram_macro_info=SRAMMacroInfo("TS1N16FFCLLSBLVTC512X32M4S"),
```

Comment:
Why we need this to be set?

Reply:
It doesn't need to be - since we generate with a stub anyway :/

Reply:
Can you remove it since it's not necessary?